website/docs/guides/dbt-ecosystem/databricks-guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md

```diff
@@ -0,0 +1,185 @@
+# How to use Databricks workflows to run dbt Cloud Jobs
+
+## Why
+
+Using Databricks workflows to call the dbt Cloud Job API can be useful for a few reasons:
+
+1. **Integration with other ETL processes**: If you have other ETL processes running in Databricks, you can use a Databricks workflow to trigger a dbt Cloud job once the ETL processes have completed.
+2. **Utilises dbt Cloud Jobs features:** dbt Cloud gives the ability to monitor jobs, manage historical logs and documentation, optimise model timing and much more.
+3. **Separation of concerns:** detailed logs for dbt jobs will be stored in the dbt Cloud environment which means more modularity and efficient debugging. You will be able to isolate the bugs faster whilst still being able to see the overall status in Databricks.
```

Comment:
sugg diff title, does 'detailed logging' encompass what you meant?
```suggestion
3. **Detailed logging:** Detailed logs for dbt jobs in the dbt Cloud environment can lead to more modularity and efficient debugging. By doing so, it becomes easier to isolate bugs quickly while still being able to see the overall status in Databricks.
```

Reply:
Seperation of concerns is a software engineering phrase: https://en.wikipedia.org/wiki/Separation_of_concerns

Reply:
oh gotcha, ok cool thank you for clarifying that Mark! no def keep it and I'll sort that for you!